data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -170,7 +170,7 @@
 NPL_FIELD = "Proximity to NPL sites"
 AIR_TOXICS_CANCER_RISK_FIELD = "Air toxics cancer risk"
 RESPIRATORY_HAZARD_FIELD = "Respiratory hazard index"
-UST_FIELD = "Underground storage tanks"
+UST_FIELD = "Leaky underground storage tanks"
```

Comment:
@emma-nechamkin and @KameronKerger - can these also be used for the sidepanel copy?